### DISCUSSION TITLE
next release

### DISCUSSION THREAD

**Discussion by machour** [avalanche123/Imagine#804] [General]
Hey,

Noticed you've been busy with this repository lately, and I was wondering if you have a date in mind for the next release?

I'm particularly interested in the added `heic` support.
Would gladly help if there was a clear task list/roadmap available somewhere.

And last but not least, thank you for this package, makes images manipulations a breeze!

**Comment by mlocati** [comment]
A new release will be published when these issues will be fixed: https://github.com/php-imagine/Imagine/milestone/1 (help welcome of course)

**Comment by mlocati** [comment]
> I'm particularly interested in the added `heic` support.

@machour In any case, please remark that you'll need an imagick PHP extension compiled against a version of ImageMagick supporting heic ([here's an (untested) example about how to do that for Ubuntu 18.04](https://eplt.medium.com/5-minutes-to-install-imagemagick-with-heic-support-on-ubuntu-18-04-digitalocean-fe2d09dcef1)).
